app/views/catalog/_form_server_profile_template_options.html.haml

```diff
@@ -0,0 +1,15 @@
+- url = url_for(:id => "#{@edit[:rec_id] || "new"}", :action => "atomic_form_field_changed")
```

Comment:
Lint error can be fixed like this - 
`- url = url_for(:id => (@edit[:rec_id]&.to_s || "new"), :action => "atomic_form_field_changed")
`